meson.build

```diff
@@ -33,27 +33,39 @@ else
 	add_project_arguments('-DHAVE_ELOGIND=1', language : 'c')
 endif
 
+gdk = dependency('gdk-3.0', required: get_option('icon'))
+gdk_pixbuf = dependency('gdk-pixbuf-2.0', required: get_option('icon'))
+icon_feature = not get_option('icon').disabled()
+if icon_feature
```

Comment:
This won't work if `-Dicon=auto` and gdk-pixbuf isn't found. Instead, the `if` should be changed to:

```meson
if gdk.found() and gdk_pixbuf.found()
```

Reply:
This has been discussed a bit back and forth (see https://github.com/emersion/mako/pull/115#discussion_r257512600). What is the wanted behavior? As soon as I know that for sure I can implement it. Is it:

    has dep | -Dicon   | outcome
    ----------------------------------
    yes     | enabled  | with icons
    no      | enabled  | error?
    yes     | auto     | with icons
    no      | auto     | without icons
    yes     | disabled | without icons
    no      | disabled | without icons

Reply:
Meson features will make the build fail if `-Dicons=enabled` and GDK isn't available. This is what `required: get_option('icon')` does. More info: http://mesonbuild.com/Build-options.html#features

Reply:
By the way, nitpick: I'd rename the `icon` option to `icons`, for consistency.

Reply:
Got it. Haven't worked with meson much outside some trivial projects, so it's a learning experience.